aipal/resources/migrations/20210215000000-ohjauskysely.up.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO kyselytyyppi (id, nimi_fi) VALUES ('ohjaus', 'Ohjaus');
```

Comment:
Pitäisikö tämä nykyään lisätä myös export skeemaan?

Reply:
ei välttämättä vielä tarvetta kun näitä ei tarjota kenellekäään tai Vipuseen.

Reply:
Hyvä jos ei palauteta vielä mistään.